Slider/Assets/Scripts/UI/Artifact/Tabs/MagiTechTabManager.cs

```diff
@@ -30,4 +31,31 @@ public void PreviewOnHoverExit()
         previewTabAnimator.SetBool("isHovered", false);
         previewTabAnimator.SetFloat("speed", previewTabAnimator.GetFloat("speed") * -1);
     }
+
+    private InputAction altViewHoldAction;
+
+    private void OnEnable()
+    {
+        altViewHoldAction = Player.GetInstance().GetComponent<PlayerInput>().actions.FindAction("AltViewHold");
```

Comment:
we have a wrapper for binding actions that travis wrote so prob try to refactor this to use it for consistency? you can ask him or look at `CameraDolly.cs` for an example

Reply:
something something `Controls.AddBindingBehavior` idk ask travis wah